src/dft/descriptor.cxx

```diff
@@ -139,8 +139,8 @@ descriptor<prec, dom>::descriptor(std::vector<std::int64_t> dimensions) {
     values_.bwd_scale = 1.0;
     values_.fwd_scale = 1.0;
     values_.number_of_transforms = 1;
-    values_.fwd_dist = 1;
-    values_.bwd_dist = 1;
+    values_.fwd_dist = 0;
+    values_.bwd_dist = 0;
```

Comment:
The spec if fairly clear that these values should be 1. What is the issue with them being 1? The tests still pass

Reply:
Sorry Anant, I forgot to run the descriptor test as you suggested. Here is a diff for the fix we discussed 
```diff
diff --git a/tests/unit_tests/dft/source/descriptor_tests.cpp b/tests/unit_tests/dft/source/descriptor_tests.cpp
index eefe460..456968f 100644
--- a/tests/unit_tests/dft/source/descriptor_tests.cpp
+++ b/tests/unit_tests/dft/source/descriptor_tests.cpp
@@ -432,11 +432,11 @@ inline void recommit_values(sycl::queue& sycl_queue) {
         { std::make_pair(config_param::LENGTHS, std::int64_t{ 10 }),
           std::make_pair(config_param::FORWARD_SCALE, PrecisionType{ 1.2 }),
           std::make_pair(config_param::BACKWARD_SCALE, PrecisionType{ 3.4 }) },
-        { std::make_pair(config_param::NUMBER_OF_TRANSFORMS, std::int64_t{ 5 }),
-          std::make_pair(config_param::COMPLEX_STORAGE, config_value::COMPLEX_COMPLEX),
+        { std::make_pair(config_param::COMPLEX_STORAGE, config_value::COMPLEX_COMPLEX),
           std::make_pair(config_param::REAL_STORAGE, config_value::REAL_REAL),
           std::make_pair(config_param::CONJUGATE_EVEN_STORAGE, config_value::COMPLEX_COMPLEX) },
         { std::make_pair(config_param::PLACEMENT, config_value::NOT_INPLACE),
+          std::make_pair(config_param::NUMBER_OF_TRANSFORMS, std::int64_t{ 5 }),
           std::make_pair(config_param::INPUT_STRIDES, strides.data()),
           std::make_pair(config_param::OUTPUT_STRIDES, strides.data()),
           std::make_pair(config_param::FWD_DISTANCE, std::int64_t{ 60 }),
```

Reply:
this is more of a workaround and the real problem could be better fixed if we let the values be 0, and with that the original descriptor tests work fine. I will begin work on changing the specs

Reply:
Until the spec has changed these values must be 1.

Reply:
Also since this does remove an invalid state, it might be best to make this change anyway.

Reply:
Even though we agreed the value could be the true i.e. `0` in the meeting, I am willing to change this, because the test is nonsensical if distances are checked before setting. eea2936809f3129ce13cd1a749a1e29739f23257

Reply:
I believe we agreed that there is no good default value for the distances. You need to remember that this interface has is supposed to supported many backends, not just MKLCPU.